heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/service/OAuthInterceptorService.java

```diff
@@ -88,6 +88,8 @@
 
     private RequestContext context;
 
+    private static final String BEARER = "bearer";
```

Comment:
Put this constante together with others (just below `TIME_REFRESH_TOKEN`)

Reply:
Done

Reply:
Done